### PR TITLE
More options

### PR DESCRIPTION
**Pull request by aqsasaied**
sc
Screenshot_20221103-152515_Expo Go
Screenshot_20221103-152522_Expo Go

### CODE REVIEW

File: src/screens/select.tsx [hunk 1]
```diff
@@ -1,51 +1,12 @@
-/* eslint-disable @typescript-eslint/no-misused-promises */
```
Comment (biadars) [inline src/screens/select.tsx:1]: This is pretty huge, I think you should split it up into some subcomponents. For example you could have a component for increasing/decreasing a number with a label, and use it twice here, one for adults and one for children. You could also wrap your datepicker into a component and move that logic there.

File: src/screens/select.tsx [hunk 2]
```diff
@@ -95,11 +75,76 @@ const SelectScreen: React.FC<SelectScreenProps> = ({ navigation }) => {
...
+            journeysDetails: response.outboundJourneys,
+          });
+        }
+      })
+      .catch((error) => {
+        console.log('Oops', error);
+        setStatusMessage(JSON.stringify(error));
+      });
+  };
+  const incAdult = () => {
```
Comment (biadars) [inline src/screens/select.tsx:109]: I'm not a huge fan of abbreviations in variable and method names - increase and decrease aren't that much longer and read a bit nice

**Review by biadars — COMMENTED** [review]
This looks like it has a lot of flexibility in terms of what the user can search for, well done!

File: package.json [hunk 3]
```diff
@@ -23,21 +23,22 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-native": "0.68.2",
+    "react-native-calendar-picker": "^7.1.2",
     "react-native-dropdown-select-list": "^1.0.23",
     "react-native-gesture-handler": "~2.2.1",
-    "react-native-material-dropdown": "^0.11.1",
-    "react-native-modal-dropdown": "^1.0.2",
-    "react-native-paper": "^4.12.1",
+    "react-native-increment-decrement-button": "^1.0.0",
```
Comment (biadars) [inline package.json:29]: Just noticed as well - is this used?

File: src/screens/select.tsx (hunk 1 above)
Comment (aqsasaied) [reply src/screens/select.tsx:1]: Will do this in the next refactor

**Pull request merged**